lib/generators/spree_stock_notifications/install/templates/spree_stock_notifications.rb

```diff
@@ -1,7 +1,11 @@
 Spree.config do |config|
   # create a list of admin emails
-  admin_role   = Spree::Role.find_by_name('admin')
-  admin_emails = admin_role.users.map(&:email).join(',')
+  if ENV['RAILS_ENV'] == 'test'
```

Comment:
Hey @farneman, 

Feel its better to not add this to the initializer, since all sites will then have this code. Would prefer to do it in a `before` block in the specs.